test/tests/functional/pbs_cgroups_hook.py

```diff
@@ -172,12 +172,14 @@ def setUp(self):
             self.eatmem_script + 'EOF\n'
         self.eatmem_job2 = \
             '#PBS -joe\n' + \
-            'sleep 2\n' + \
-            'python - 200 10 <<EOF\n' + \
+            'let i=0; while [ $i -lt 400000 ]; do let i+=1 ; done\n' + \
+            'python - 200 2 <<EOF\n' + \
             self.eatmem_script + 'EOF\n' + \
-            'sleep 5\n' + \
-            'python - 100 10 <<EOF\n' + \
-            self.eatmem_script + 'EOF\n'
+            'let i=0; while [ $i -lt 400000 ]; do let i+=1 ; done\n' + \
+            'python - 100 4 <<EOF\n' + \
+            self.eatmem_script + 'EOF\n' + \
+            'let i=0; while [ $i -lt 400000 ]; do let i+=1 ; done\n' + \
+            'sleep 5\n'
```

Comment:
The reason this job script was updated to allocate less memory as sometime VMs were running out of memory or causing a hang. Why do we need this job script?

Reply:
It allocates less memory than it used to, and is need to accumulate CPU time so that the measurements get larger as the job runs.

Reply:
Thanks Mike. Please update the script to run explicitly with bash. As these syntax are failing with csh.